extension/src/popup/views/GrantAccess.tsx

```diff
@@ -66,21 +100,38 @@ export const GrantAccess = () => {
     <GrantAccessEl>
       <HeaderEl>Connection request</HeaderEl>
       <SubheaderEl>{punycodedDomain} wants to know your public key</SubheaderEl>
+      {!isDomainWhiteListed ? (
+        <WarningMessageEl>
+          <p>
+            <strong>
+              This is the first time you interact with this domain in this
+              computer.
```

Comment:
Grammar on this is funky, maybe
```suggestion
              This is the first time you‘ve interacted with this domain on this
              computer.
```